### PR TITLE
Client side validation: The content of the email field has to be in lower case

### PR DESCRIPTION
**Pull request by veronica365**
#### Description
- When the user submits the form, you check if the email is in lower case.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -200,3 +200,50 @@ allLinks.forEach((link) => {
...
+  );
+  if (!isValidEmail) {
+    error = 'Error: Ensure that you have typed the correct email';
+    return setFormMessage(error);
+  }
+
+  if (email !== String(email).toLowerCase()) {
+    error = 'Error: Ensure that the email is typed in lowercase';
+    return setFormMessage(error);
+  }
```
Comment (BenMukebo) [inline assets/js/script.js:242]: - [ ] Great job of adding validation by checking if the email is written in lowercase 👏🏻 However, I suggest you change the color of the text error message to something stronger. You must use a color that can sound more error 🤓 and not white, but make sure the error message matches the chosen template design in Figma (e.g. font, colors, images, tests, margins).

Screenshot from 2023-02-03 05-45-36

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Team,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters is set correctly project repository ✔️ 
- The error message should show up close to the `submit` button.  ✔️   
- The form can check if the email is all lowercase ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

``` ## Authors ```

- [ ] Update your Readme, This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 👨🏻‍🎨 

- The README file is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by veronica365** [comment]
- All feedback has been implemented

File: styles.css [hunk 2]
```diff
@@ -103,6 +103,19 @@ header.toolbar img {
...
+small {
+  display: none;
+}
+
+small.form-message {
+  display: flex;
+  color: #f00;
+  margin-bottom: 1rem;
+  justify-content: center;
+  background-color: #ebe7e0;
```
Comment (pinkmoon25) [inline styles.css:115]: - [ ] [OPTIONAL] Good job implementing the changes requested by the previous reviewer :clap:  I would suggest to drop the background color.

**Review by pinkmoon25 — APPROVED** [review]
Hi @team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Changes implemented :heavy_check_mark: 
Readme :heavy_check_mark: 
Github flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**